whyerror/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
whyerror/cli.py,sha256=2jw9EUv-wMYm0qpLexbuDVEmm25GXsomaBSFn8BzBw8,2513
whyerror-0.0.7.dist-info/LICENSE,sha256=E9EzhV43t6p4nIvGfM8qWh3TW71_DC22Klq6SxQBhlI,1009
whyerror-0.0.7.dist-info/METADATA,sha256=x1jvYruVkpvZN4HqFvXsT8Q_haGPgh96BrP8mvKF8n0,192
whyerror-0.0.7.dist-info/WHEEL,sha256=pkctZYzUS4AYVn6dJ-7367OJZivF2e8RA9b_ZBjif18,92
whyerror-0.0.7.dist-info/entry_points.txt,sha256=emqedR3iw5ohTm0XmMU71v1ZGofCr7me1g1BunLGLsk,51
whyerror-0.0.7.dist-info/top_level.txt,sha256=ezq_loM7TfcW-39u3pzZZu_WdTe8QeNR53wRvyqCJwI,9
whyerror-0.0.7.dist-info/RECORD,,
